summoner-cli/src/Summoner/Project.hs

```diff
@@ -311,10 +311,13 @@ doGithubCommands Settings{..} = do
     runGh :: Text -> IO Bool
     runGh repo =
         True <$ "gh" (["repo", "create", "-y", "-d", settingsDescription, repo]
-             ++ ["--private" | settingsPrivate])  -- Create private repository if asked so
```

Comment:
I feel bad for removing this and replacing it for an if-then-else. Looked so elegant! Perhaps you know any other way? I'm not that knowledgeable about Haskell (yet!)